scripts/fedora/fix-slow-dns.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -eux
+
+if [[ "$PACKER_BUILDER_TYPE" == virtualbox* ]]; then
```

Comment:
Are we planning to build virtualbox images?

Reply:
Nope, so we can remove this.